packages/core/types/bugsnag.d.ts

```diff
@@ -0,0 +1,66 @@
+import Client from "./client";
+import Event from "./event";
+import {
+  BreadcrumbType,
+  BreadcrumbMetadataValue,
+  Config,
+  NotifiableError,
+  OnErrorCallback,
+  OnSessionCallback,
+  OnBreadcrumbCallback,
+  User,
+  Plugin
+} from "./common";
+
+export default interface BugsnagStatic {
```

Comment:
Consider using `export default interface BugsnagStatic extends Client` to save having to restate the `Client` type signature.